dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/organisationunit/DefaultOrganisationUnitService.java

```diff
@@ -443,21 +457,21 @@ public OrganisationUnitDataSetAssociationSet getOrganisationUnitDataSetAssociati
     }
 
     @Override
-    @Transactional(readOnly = true)
+    @Transactional( readOnly = true )
     public List<OrganisationUnit> getOrganisationUnitsBetweenByName( String name, int first, int max )
     {
         return organisationUnitStore.getAllLikeName( name, first, max );
     }
 
     @Override
-    @Transactional(readOnly = true)
+    @Transactional( readOnly = true )
     public boolean isInUserHierarchy( OrganisationUnit organisationUnit )
     {
         return isInUserHierarchy( currentUserService.getCurrentUser(), organisationUnit );
     }
 
     @Override
-    @Transactional(readOnly = true)
+    @Transactional( readOnly = true )

```

Comment:
I know the code style might be ambiguous but please don't reformat entire classes just to add a single method.